runtime/stdlib/test_test.go

```diff
@@ -2084,25 +2091,27 @@ func TestBlockchain(t *testing.T) {
 	t.Run("typed events, empty", func(t *testing.T) {
 		t.Parallel()
 
-		script := `
-		    import Test
+		const script = `
+            import Test
 
-		    pub fun test(): [AnyStruct] {
-		        var blockchain = Test.newEmulatorBlockchain()
+            pub struct Foo {}
+
+            pub fun test() {
+                let blockchain = Test.newEmulatorBlockchain()
 
-		        // 'Foo' is not an event-type.
-		        // But we just need to test the API, so it doesn't really matter.
-		        var typ = Type<Foo>()
+                // 'Foo' is not an event-type.
+                // But we just need to test the API, so it doesn't really matter.
+                let typ = Type<Foo>()
 
-		        return blockchain.eventsOfType(typ)
-		    }
+                let events = blockchain.eventsOfType(typ)
 
-		    pub struct Foo {}
+                Test.expect(events, Test.beEmpty())
+            }
 		`
 
 		eventsInvoked := false
 
-		testFramework := &mockedTestFramework{
+		emulatorBackend := &mockedTestFramework{
```

Comment:
Here and below: Maybe also move this into `newEmulatorBackend`, so a new `&mockedTestFramework` is created on each call of the `newEmulatorBackend` function

Reply:
Ah yes, good suggestion :100: 
Now it's the same instance again :sweat_smile: